type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_distinct_array: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_timestamp_builtins: Name 'timestamp_cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_decimal_builtins: Name 'general_cases' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_parse_url: No attribute '_check_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_histogram_value_counts: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_timestamp_scalar_in_filter: Name 'expr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_aggregations: Name 'exprs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in find_by_value: Name 'elements' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in click: Name 'parent_form' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in prev: Name 'prev_index' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in close: No attribute '_browser' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in close_others: Name 'window' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in read: Name 'fid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _get_aws_auth_param: Function hmac.new was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _get_boolean: No attribute 'findtext' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __iter__: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in find_by_xpath: Name 'elements' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in find_by_tag: Name 'elements' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _do_PF_SKIP: Name 'n' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _do_PF_INS_BZ2: No attribute 'outfile' on bytes [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in _do_PF_BSDIFF4: Name 'source' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _do_PF_REC_ZIP: Name 'workdir' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in _diff_file: Name 's_mod' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in chainload: Name 'bsfile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 57, in read: Name '_size14' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in find_option_by_value: Name 'control' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in minify: Name 'minifiable_status' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: No attribute '_copy_data_to_form' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in teardown: Name 'todo' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in decimal_table: No attribute 'execute' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __repr__: Name 'kvs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in async_refresh: No attribute 'task_options' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in expiry: No attribute 'lifetime' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in timeout: No attribute 'refresh_timeout' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in should_missing_item_be_fetched_synchronously: No attribute 'fetch_on_miss' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in key: No attribute 'hash' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_cache: Name 'cache' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _CloneEntryCB: Name 'entry_month' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _DeleteEntryCB: Name 'entry_day' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _TreeExpandMenu: Name 'tree' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _FileRevertMenu: Name 'day' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in OnCompareItems: Name 'data2' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in Prune: Name 'parent_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in split_app_version: Name 'platform' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _split_version_components: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 8, in get_original_filename: Name 'filtered' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in diff: Function str.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in patch: No attribute 'append' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _write_zipfile_metadata: Function ZipFile.writestr was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 33, in paths_differ: Name 'nm' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 18, in calculate_digest: No attribute 'digest' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in unlock: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in needs_cleanup: Name 'act' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in id_generator: Function random.choice was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in validate: Name 'initial_validation' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_string_to_integer: No attribute 'string_to_integer' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in testBrokenIsDstHandling: 'datetime' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in testGMTOffset: 'datetime' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get: No attribute 'response' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in get: Name 'qr_img' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in printer: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in read_until: Name 'escaped_char' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in testContains: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testCachePre: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in testSetExDateRevOrder: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 21, in testSetDateAndExDate: 'datetime' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in testFuzzy: No attribute 'assertEqual' on str [attribute-error]
File ""dummpy_input_file"", line 5, in testFuzzy: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in testMicrosecondPrecisionErrorReturns: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 9, in testCustomParserInfo: 'datetime' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in testStrStart2: No attribute 'assertEqual' on str [attribute-error]
File ""dummpy_input_file"", line 7, in testStrStart2: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testStrEnd2: No attribute 'assertEqual' on str [attribute-error]
File ""dummpy_input_file"", line 7, in testStrEnd2: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 8, in testFileStart1: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 9, in testFileEnd1: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testZoneInfoFileStart1: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testZoneInfoFileEnd1: 'datetime' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in testICalStart1: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 8, in testICalEnd1: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testRoundNonFullMinutes: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 7, in testLeapCountDecodesProperly: Name 'tz' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in run: Name 'fh' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in fetch_result: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in _diff_binary_file: unsupported operand type(s) for +: 'sfile: Optional[BinaryIO]' and 'skipbytes: int' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in _write_file_patch: No attribute '_write_bytes' on List[Tuple[int, tuple]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in __exit__: No attribute 'path' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 55, in main: No attribute 'diff_window' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in start: Name 'monitor_thread' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in __getattr__: Name 'msg' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 7, in _get_sudo_argtypes: Name 'argtypes' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in _get_sudo_iterator: Name 'iterator' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _CalendarDisplayChanged: Name 'months' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in main: Name 'info_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _esky_lazy_load: Name 'ns' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in common_prefix: Name 'c2' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in appexe_from_executable: Name 'potential_py2app_bootstrap_exe' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in zipfile_common_prefix_dir: Name 'prefix' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in gen_members: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_backup_filename: Name 'parts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 34, in snapshot_single_frame: No attribute 'type' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in read: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_request_header: Name 'http_key' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in csv_to_list: unsupported operand type(s) for item retrieval: 'i: int' and 'i: int' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 7, in setup_urllib2: No attribute 'webdav' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __getstate__: Name 'state' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in extract: No attribute 'extractall' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in cleanup: Name 'ext' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in coordinates: Name 'sw' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in prepdir: Name 'path' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in append_geojson: unsupported operand type(s) for >>: 'print: Callable[[], None]' and 'out: TextIO' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in render_error: Name 'error_code' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in register_logging: No attribute 'logger' on logging.StreamHandler[TextIO] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in GetInHMS: Name 'minutes' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_connections: No attribute 'request' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _flatten: Name 'context_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in match: Name 'm' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in surlex_to_regex: Name 'scribe' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in parsed_surlex_object: Name 'object' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in instructionsMenuClicked: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: No attribute 'close' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'progress' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 60, in crawl_memory: Name 'distro' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in crawl_load: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in crawl_dockerhistory: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_errno_msg: Name 'errno' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in pop: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in x: Name 'w' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_cgroup_dir: Name 'proc' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _log_locations_json_sanity_check: No attribute 'keys' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in setup_logger: Name 'logfile_name' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in setup_logger_stdout: Function StreamHandler.setFormatter was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in main: Name 'trace_dir' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in build_from_csv: Name 'fr' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in build_from_file: Name 'call' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in pipPipImageBrowsePressed: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in isPreLinked: Name 'dyn' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in save_data: Name 'fh' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in load_log_file: No attribute 'log' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in close: Name 'mine' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _write_dests: No attribute 'output' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in SetDayAttr: Name 'attr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 71, in OnInit: Can't assign attribute 'prev_id' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _SetTitle: No attribute 'frame' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _UpdateAuthorBox: No attribute 'author_label_id' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in read_cookie: Name 'cookies' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in extend: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in send: No attribute 'mailbox' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in call: No attribute 'send' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in finalize_options: No attribute 'freezer_module' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_bootstrap_code: Name 'submod' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in add_to_bootstrap_manifest: Name 'fpath' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'parseArguments' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 29, in setup: No attribute 'videocodeccombo' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in doShow: No attribute 'Show' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in fileBrowser: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in showWarning: Name 'dlg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in execute: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_storage: No attribute 'name' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in find_video_address_pr: No attribute 'getElementsByTagName' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in find_link_you_get: No attribute 'startswith' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_resolution_mediainfo: Name 'resolution' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_url_size: Name 'site' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in convert_ass_py2: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in main: Name 'server' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in send_sequence: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in schedule: Name 'next_task' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in send_exception: No attribute '_channel' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in __init__: No attribute 'kwargs' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in make_complaints: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 9, in make_uof: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_unknown_resident_weapon_returned: No attribute 'resident_weapon_used' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_known_resident_weapons_cleaned: Name 'check_weapon' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_capitalization: Name 'send' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_captilization_handles_non_strings: Name 'result_float' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in get: No attribute 'class_path' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in delete: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in serve: Name 'server' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in insert_script: Name 'sql' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in commit: No attribute 'wait' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_getset_metadata: Name 'md' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_file_move: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in resolveExpression: No attribute 'json' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_class_members: No attribute 'Meta' on List[Tuple[str, Any]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in delete: Name 'dataset' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in upload: Name 'dir_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in write_tmp_file: Function BinaryIO.write was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in sender: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in minify: Missing parameter 'contents' in call to built-in function bytes.minify [missing-parameter]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 40, in emit: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _filetype: Name 'modebit' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _fileperm: Name 'char' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in _is_executable: Function os.access expects 2 arg(s), got 3 [wrong-arg-count]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _is_config_file: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in _crawl_processes: No attribute 'ppid' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _crawl_connections: No attribute 'status' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in crawl_metrics: Name 'feature' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 22, in _crawl_metrics: No attribute 'username' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in sender: No attribute 'send' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in testBlockTimeout: No attribute 'fail' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testPer: No attribute 'compareConversion' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in testReadability: No attribute 'compareConversion' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in testFloat: Can't assign attribute 'units' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in testToday: No attribute 'compareDate' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in testThis: No attribute 'compareDate' on datetime.datetime [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in testIllegalDate: No attribute 'assertRaises' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testTwoDates: No attribute 'compareTimes' on List[datetime.datetime] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_defaults_no_origin: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_regex_list: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_index: Name 'resp' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_wildcard: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in getContextsForExpression: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in getRetinas: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testCreateCategoryFilter: Name 'filter1' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_default: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_database_has_cycles: Name 'database' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_database_has_no_cycles_2: Name 'database' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_database_sorted_relations_cyclic: Name 'database' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_database_create_relations_script: No attribute 'script_database' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_database_drop_relations_script: No attribute 'script_database' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_database_recreate_relations_script: No attribute 'script_database' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_database_recreate_table_dependencies: No attribute 'recreate_table_dependencies' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in validate_pipeline_definition: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in text: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in update_attributes_from_columns: Name 'column' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in select_duplicates_script: Name 'pk_columns' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in test_basic: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 12, in test_exists_clone: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in read_file: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in extract_words: No attribute 'read' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in extract_words: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute '_args' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in create_db_schema: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _update_view: Can't assign attribute '_freqs_0' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: No attribute '_stop_words' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __produce_words: Name 'w' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_delete: Name 'ret' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_url: No attribute 'read' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_list: No attribute 'save' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_context_data: No attribute 'kwargs' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in rnn_output: Name 'rnn_out' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in add_test: No attribute 'run_test' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in add_row: Name 'cols' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in pointer_decl: Name 'args' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in pointer_as_arg: Name 'args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in difference_update: Name 'elem' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in intersection_update: Can't assign attribute 'elements' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in serialize_option: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_default: Name 'resp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_allow_headers_with_request_headers: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_override: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in to_inch: Name 'primitive' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in from_dict: 'int' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in from_dict: Name 'param' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in to_gerber: No attribute 'b' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: No attribute 'a' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __str__: No attribute 'b' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __exit__: No attribute 'close' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_results: No attribute 'res_queue' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in test_FSParamStmt_factory: No attribute 'notation' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_FSParamStmt: No attribute 'notation' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_MOParamStmt_conversion: No attribute 'mode' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_IPParamStmt_factory: Name 'ip' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_IPParamStmt: No attribute 'param' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 12, in test_IPParamStmt_dump: No attribute 'to_gerber' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_IRParamStmt_factory: No attribute 'angle' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_IRParamStmt_dump: Name 'ir' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_OFParamStmt_factory: No attribute 'b' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_OFParamStmt_dump: No attribute 'to_gerber' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_SFParamStmt_factory: No attribute 'a' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_SFParamStmt_dump: Name 'sf' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_SFParamStmt_string: Name 'sf' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in readint: No attribute 'eof' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in print_instructions: Name 'opcode' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in work: Name 'filename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in save: Name 'object_policy' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in delete: Name 'obj' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in parse_size: No attribute 'find' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_LPParamStmt_factory: Name 'lp' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_LPParamStmt_dump: No attribute 'to_gerber' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_reading_multiple_chunks: Name 'demuxer' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_reading_partial_chunk: Name 'demuxer' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in printable: Function Pattern.sub was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 3, in __exit__: No attribute 'stop' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_autodiscover: Name 'names' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_autodiscover_overwrite: Name 'names' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in count_check: No attribute 'destination_sql' on argparse.ArgumentParser [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in create_error_retrieval_query: Name 'condition' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in primary_key_check: No attribute 'log_to_s3' on argparse.ArgumentParser [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 22, in script_runner: Name 'input_file' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in read_input: Name 'line' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in read_mapper_output: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_AMParamStmt_dump: No attribute 'to_gerber' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_AMParamStmt_string: No attribute 'build' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_ASParamStmt_factory: No attribute 'mode' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_ASParamStmt_dump: No attribute 'to_gerber' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_INParamStmt_factory: No attribute 'name' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_INParamStmt_dump: Name 'inp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_LNParamStmt_factory: No attribute 'name' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_LNParamStmt_dump: Name 'lnp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_quadmodestmt_factory: No attribute 'type' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_quadmodestmt_dump: Name 'stmt' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_regionmodestmt_dump: Name 'stmt' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_unknownstmt: No attribute 'type' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in test_ADParamStmt_factory: No attribute 'd' on Dict[str, Union[int, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 39, in test_ADParamStmt_conversion: No attribute 'to_metric' on Dict[str, Union[int, str]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_MIParamStmt_factory: No attribute 'a' on Dict[str, Union[int, str]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in test_coordstmt_factory: No attribute 'op' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in israw: No attribute 'raw' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in attach_socket: Name 'socket' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in exec_create: Name 'exec_id' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in israw: No attribute 'raw' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in resize: Name 'rows' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _hijack_tty: No attribute 'to_stream' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_create_chat: No attribute 'assertSuccessResponse' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_create_audio_url: No attribute 'client' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in build_api_key_url: No attribute 'build_url' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_blog_posts: Name 'format_params' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in decode: Name 'decompressed' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in thread_level: No attribute 'id' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in send_message: No attribute 'send' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_create_graphitesend_instance: Name 'g_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_no_system_name: Name 'custom_prefix' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_set_prefix_group_system: Name 'custom_prefix' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in testGetCell: No attribute 'assertEqual' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in _walk_StatIf: No attribute '_walk' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in prompt_pass: Name 'rv' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in description: Name 'description' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_options: No attribute 'use_debugger' on Tuple[Any, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in init_asd_db: Name 'asd_db' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in offline_snippet_lookup: Name 'snippet' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in online_snippet_lookup: Name 'snippet' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in offline_estimated_applecare_end_date: Function dateutil.parser.parse was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in warranty_generator: unsupported operand type(s) for item assignment: None [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'stream' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 37, in main: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _walk_VarIndex: Name 't' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _walk_NameList: No attribute '_get_name' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in _walk_TableConstructor: No attribute '_args' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in testWriteQuiet: No attribute 'assertEqual' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in testError: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testDebug: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in testFromLines: No attribute '_data' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_valid_file: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_invalid_file: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_submitted_not_valid: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_hidden_tag: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_csrf_token: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in to_lines: No attribute '_data' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _any_running: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'filename' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _walk: Name 'item' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in walk: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute '_pos' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 7, in _get_code_for_spaces: No attribute '_pos' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in _get_name: Name 'spaces' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _get_semis: No attribute '_get_code_for_spaces' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _walk_StatFunctionCall: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in string2port: No attribute 'group' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_cursor_id: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_connect_exception_on_badhost: No attribute 'assertRaises' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in render_with_filter: Name 'exc' is not defined [name-error]
File ""dummpy_input_file"", line 10, in render_with_filter: 'formatter' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_create_root_folter_if_not_exists: No attribute 'opt' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_create_pid_structure: No attribute 'opt' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_create_log_dir: No attribute 'opt' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_create_app_dir: No attribute 'opt' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 25, in test_init_an_already_inited_path: No attribute 'assertTrue' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in setUp: Can't assign attribute 'config' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_create_all_options: Name 'opt' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_validate_recv: Name 'opt' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in setUp: Can't assign attribute 'init' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_invalid_pid_files: No attribute 'init' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in setUp: Can't assign attribute 'opt' on tuple [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in test_check_pids_are_running: No attribute 'assertEquals' on str [attribute-error]
File ""dummpy_input_file"", line 25, in test_check_pids_are_running: No attribute 'call_count' on Callable [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_start_response_app_return_iterable: No attribute 'assertEquals' on List[Tuple[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in get_include_file_content: Name 'file_to_read' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in delete_temp_files: Name 'temp_file' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_create_quote: Name 'json_response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_create_link: No attribute 'client' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in run_test: No attribute 'recv' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in to_seqfeature: Name 'qualifiers' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in derived_feature_generator: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _write_object: Name 'f' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _read_object_gen: Name 'filename' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in delete: Name 'path' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in contains: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_app_info: Name 'app_info' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_app_health: Name 'app_health' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_login: No attribute 'status_code' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in insert: No attribute 'table_name' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in insert_many: Name 'conn' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in query: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in outputmap: Name 'map_target' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in path: No attribute 'strip' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in locatePlugins: No attribute 'recursive' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in setVersion: No attribute 'details' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in setPluginInfoExtension: Can't assign attribute 'expectedExtensions' on Tuple[Any] [not-writable]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _extractCorePluginInfo: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in isValidPlugin: Name 'reg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'dst_port' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'vid' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'ip' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'cost' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 's_dpid' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'mac' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 's_port' on Dict[str, Any] [not-writable]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'ts' on Dict[str, Any] [not-writable]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'topic' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'role' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'ip' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in locatePlugins: Name 'npc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in activatePluginByName: Name 'pta_item' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in get: Name 'pm' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in create_schema: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'name' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Can't assign attribute 'persistence' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'icmpv6' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'eth_dst' on Dict[str, Any] [not-writable]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'icmp_code' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'ip_dscn' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'target_address' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'options' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'name' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'app_id' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in dataModel: Name 'model' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in parse: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in detect_excellon_format: No attribute 'comment' on Tuple[str, str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in function_line: Name 'line' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_name: Function Pattern.match was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_lorem: Function Pattern.match was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 8, in test_address: No attribute 'match' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_phonenumber: Function Pattern.match was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 8, in test_company: No attribute 'match' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_name_multiple: Name 'v1' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _get_value: Name 'attr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_object_directory: Name 'module' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_path_matches: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in read_fixture_files: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 5, in step_install_cli: Name 'dists' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_dont_strip_leading_whitespace: 'tabulate' object is not callable [not-callable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in drop_tables: Name 'cur' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute '_sockets' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in on_stop: No attribute 'stop' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_upper: Name 'reply' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_dispatch_not_found: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in load_yaml: Missing parameter 'Loader' in call to function yaml.load [missing-parameter]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in find_object: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test___all__: No attribute 'assertEqual' on Set[str] [attribute-error]"
